code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -385,14 +385,21 @@
 		if(src.loc == summoner)
 			to_chat(src, "<span class='danger'><B>You must be manifested to heal!</span></B>")
 			return
+		if(issythetic(target))
+			src.changeNext_move(CLICK_CD_MELEE)
+			if(heal_cooldown <= world.time && !stat)
+				var/mob/living/synthetic/I = target
+				if (I && I.damage >= 0)
+					I.damage = -10
+					heal_cooldown = world.time + 20
```

Comment:
No, this won't fix it. I'm not even sure we HAVE a `/mob/living/synthetic` mob! To ACTUALLY fix it, you'll want to call `heal_overall_damage(amount, amount, 0, 1)` - which will fix both burn and brute damage, so you'll still want code to mend toxins and oxy damage.